bigearthnet_encoder/encoder.py

```diff
@@ -155,6 +170,17 @@ def write_S2_lmdb_raw(ben_s2_directory_path: Path, **kwargs):
     return write_S2_lmdb(ben_s2_directory_path, **kwargs)
 
 
+@fc.delegates(write_S2_lmdb, but=["patch_path_to_metadata"])
+def write_S1_S2_lmdb_raw(ben_s1_directory_path: Path,
+                         ben_s2_directory_path: Path,
+                         **kwargs):
+    """
+    Write an S2 lmdb file that only includes the patch name
+    as the key and the patch array information as the value.
```

Comment:
Would add to the documentation that the patch array now combines both sentinel images

Reply:
Sorry, I forgot that initially. Should be added now.